rotkehlchen/tests/api/test_history_base_entry.py

```diff
@@ -527,6 +529,27 @@ def test_get_events(rotkehlchen_api_server: 'APIServer'):
                 entries_limit=limit,
             ) == (total, found)
 
+    # test query an event with an unknown asset
+    def mock_check_asset_existence(identifier: str, query_packaged_db: bool = True):
+        raise UnknownAsset(identifier)
+
+    with patch(
+        target='rotkehlchen.assets.asset.AssetResolver.check_existence',
+        new=mock_check_asset_existence,
+    ):
+        response = requests.post(
+            api_url_for(
+                rotkehlchen_api_server,
+                'historyeventresource',
+            ),
+        )
+    assert_proper_sync_response_with_result(response)
+    assert rotkehlchen_api_server.rest_api.rotkehlchen.msg_aggregator.consume_errors() == [
+        "Failed to deserialize history event (1, 6, 'STARK-STARK-STARK', 0, 1673146287380, 'B', 'Kraken', 'ETH2', '0.0000400780', '0.051645312360', 'Staking reward from kraken', 'staking', 'reward', None, None, None, None, None, None, None) due to Unknown asset ETH2 provided. Try redecoding the event or adding the asset manually.",  # noqa: E501
```

Comment:
This part of the error `history event (1, 6, 'STARK-STARK-STARK', 0, 1673146287380, 'B', 'Kraken', 'ETH2', '0.0000400780', '0.051645312360', 'Staking reward from kraken', 'staking', 'reward', None, None, None, None, None, None, None)` is not useful for the user and add noise. Consider that we display errors in a small notification and the action that the user needs to perform probably will be dismissed